lib/facilities/drivetime_band_client.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module Facilities
+  # Core class responsible for api interface operations
+  class DrivetimeBandClient < Common::Client::Base
+    configuration Facilities::DrivetimeBandConfiguration
+
+    def get_all_drivetime_bands(offset, limit)
```

Comment:
Should we still call this `get_all_drivetime_bands` considering that it now takes an offset and a limit? Maybe it should just be `get_drivetime_bands`? Is there some other piece that will call this in the future and iterate through all of them?

Reply:
Good catch! I will rename it. My half-baked plan was to give the future sidekiq download job the responsibility of iterating through all the bands.